### PR TITLE
Adding route server support

### PR DESCRIPTION
**Pull request by mwichtlh**
Hi all,

after some discussion with Jenny and Arturo, I have prepared this PR to add necessary fields for route server support at IXPs to the draft. I am happy to work on any feedback.

I was a bit unsure where to add the explanation of new fields, that are not yet reflected in the autopeer OpenAPI definition. For the time being I placed them in the BGP Session section, but they might fit better somewhere else. To be discussed.

Regards,
Matthias

### CODE REVIEW

File: draft-ramseyer-grow-peering-api.md [hunk 1]
```diff
@@ -402,12 +416,24 @@ BGP Session
...
+  * local_monitoring_session (optional, to support monitoring systems, defaults to false)
+  * peer_monitoring_session (optional, to support monitoring systems, defaults to false)
   * peer_type (public or private)
-  * md5 (optional, as defined above)
+  * session_secret (optional, as defined above)
   * location (Peering Location, as defined above)
   * status (Session Status, as defined above)
   * session_id (of individual session and generated by the server)
 
+As not all elements are reflected in the {{autopeer}} OpenAPI definition to date, we define the missing fields here to be reflected in {{autopeer}} in the future.
```
Comment (jramseyer) [inline draft-ramseyer-grow-peering-api.md:431]: This looks good.  

As an aside, we should probably update the OpenAPI spec and move it in to the IETF draft as well (also brought up in https://github.com/bgp/draft-ietf-peering-api/pull/30), but that's outside the scope of this PR.

File: draft-ramseyer-grow-peering-api.md [hunk 2]
```diff
@@ -636,19 +662,13 @@ The API described in this document follows REST ({{rest}}) principles over an HT
...
-
 IANA Considerations     {#iana}
 ===================
 
 This document has no IANA actions.
 
 --- back
 
-
 Acknowledgments     {#acknowledgments}
```
Comment (jramseyer) [inline draft-ramseyer-grow-peering-api.md:672]: Want to add yourself as well?

**Review by jramseyer — COMMENTED** [review]
Looks good to me!  If no further comments, I'll merge it in on Wednesday.

File: draft-ramseyer-grow-peering-api.md (hunk 1 above)
Comment (mwichtlh) [reply draft-ramseyer-grow-peering-api.md:431]: > As an aside, we should probably update the OpenAPI spec and move it in to the IETF draft as well (also brought up in https://github.com/bgp/draft-ietf-peering-api/pull/30), but that's outside the scope of this PR.

I think that’s a very reasonable approach. RFCs should be self-contained. I’m happy to help with editing tasks like this.

File: draft-ramseyer-grow-peering-api.md (hunk 2 above)
Comment (mwichtlh) [reply draft-ramseyer-grow-peering-api.md:672]: Thanks, done.

**Comment by mwichtlh** [comment]
> Looks good to me! If no further comments, I'll merge it in on Wednesday.

Fine for me, thank you.

File: draft-ramseyer-grow-peering-api.md (hunk 1 above)
Comment (jramseyer) [reply draft-ramseyer-grow-peering-api.md:431]: Thanks!  We'd welcome the help.

**Pull request merged**